tests/network/vmi_lifecycle.go

```diff
@@ -60,6 +61,9 @@ var _ = SIGDescribe("[crit:high][arm64][vendor:cnv-qe@redhat.com][level:componen
 			DescribeTable("[Serial]VMIs shouldn't fail after the kubelet restarts", func(bridgeNetworking bool) {
 				var vmiOptions []libvmi.Option
 
+				arch := testsuite.TranslateBuildArch()
+				checks.SkipIfS390X(arch, "failing for s390x.")
```

Comment:
Please don't use skips, also how does this fail?

Reply:
@xpivarc do we have an example here how to create a label for the tests?

Reply:
The labels are defined [here](https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go). We should use these labels to filter what tests will run for the s390x lane. If for whatever reason we need to skip some individual test (`It`) we should not label the `Describe` but lower leafs `Context` or `It`...  to prevent the programmatic skip